torch/distributed/fsdp/_flat_param.py

```diff
@@ -2203,6 +2213,8 @@ def _writeback_orig_params(self) -> bool:
             if self.uses_sharded_strategy or not self._offload_params
             else flat_param._cpu_grad
         )
+        from torch.distributed._tensor import DTensor
```

Comment:
I think we cannot import `DTensor` at top level or else it breaks internal torch packages or something.